README.md

```diff
@@ -25,25 +25,33 @@ Even if some checkbox is marked it does not mean that feature works perfectly.
 Some of those features are a likely outside the scope of this project, we are
 still learning and exploring what's possible.
 
-- [x] Linting with Scalafix on compile (textDocument/publishDiagnostics)
-- [ ] Linting with Scalafix as you type (textDocument/publishDiagnostics)
-- [ ] Refactoring with Scalafix (textDocument/codeAction)
-- [x] Formatting with Scalafmt (textDocument/formatting)
-- [ ] Formatting with Scalafmt for range (textDocument/rangeFormatting)
-- [ ] Formatting with Scalafmt as you type (textDocument/onTypeFormatting)
-- [x] Auto completions as you type with presentation compiler (textDocument/completions)
-- [x] Show type at position as you type (textDocument/hover)
-- [x] Go to definition from project Scala sources to project Scala sources on compile (textDocument/definition)
-- [x] Go to definition from project sources to Scala dependency source files on compile (textDocument/definition)
-- [x] Go to definition from project sources to Java dependency source file on compile (textDocument/definition)
-- [ ] Go to definition as you type (textDocument/definition)
-- [x] Show parameter list as you type (textDocument/signatureHelper)
-- [ ] Show red squigglies as you type (textDocument/publishDiagnostics)
-- [ ] Show red squigglies on compile (textDocument/publishDiagnostics)
-- [ ] Auto-insert missing import when completing a global symbol (textDocument/completions)
-- [ ] Find symbol references (textDocument/references)
-- [ ] Rename local symbol (textDocument/rename)
-- [ ] Rename global symbol (textDocument/rename)
+* Formatting with Scalafmt:
+  - [x] Whole file (`textDocument/formatting`)
+  - [ ] Selected range (`textDocument/rangeFormatting`)
+  - [ ] As you type (`textDocument/onTypeFormatting`)
+* Completions and signature help:
```

Comment:
"IntelliSense with presentation compiler"?

Reply:
IntelliSense is a Visual Studio feature name, isn't it? "Intelligent code completion" seems to be a general/neutral way to call it. Or about "Context-aware code completion"?